cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/preview/DefaultPreviewManager.java

```diff
@@ -142,6 +143,8 @@
   private PreviewDataSubscriberService dataSubscriberService;
   private PreviewTMSLogSubscriber logSubscriberService;
   private LogAppender logAppender;
+  private final Double maxNumberOfRecords;
```

Comment:
Why is it a double? Should it be just an int?

Reply:
The request had the number of preview records of type Double so I used the same